src/main/asciidoc/analytics/twitter-analytics/main.adoc

```diff
@@ -37,12 +46,12 @@ Created new stream 'tweets'
 ```
 +
 ```
-dataflow:>stream create tweetlang  --definition ":tweets.twitterstream > field-value-counter --fieldName=lang --name=language" --deploy
+dataflow:>stream create tweetlang  --definition ":tweets.twitterstream > counter --name=language --counter.tag.expression.lang=#jsonPath(payload,'$..lang')" --deploy
 Created and deployed new stream 'tweetlang'
 ```
 +
 ```
-dataflow:>stream create tagcount --definition ":tweets.twitterstream > field-value-counter --fieldName=entities.hashtags.text --name=hashtags" --deploy
+dataflow:>stream create tagcount  --definition ":tweets.twitterstream > counter --name=hashtags --counter.tag.expression.htag=#jsonPath(payload,'$.entities.hashtags[*].text')" --deploy
```

Comment:
Same as above. It'd be good to add a sample JSON to show the `hashtags` data structure in the JSON.